gen/flights/waypointbuilder.py

```diff
@@ -55,7 +56,8 @@ def __init__(
 
     @property
     def is_helo(self) -> bool:
-        return getattr(self.flight.unit_type, "helicopter", False)
+        return self.flight.unit_type.dcs_unit_type.helicopter
```

Comment:
I'm a little uncertain what the most sensible way is here. This appears to work though.

Reply:
This is right. Could add a `helicopter` property to `AircraftType` if you want. Delete the commented out line though.